test/parallel/test-intl.js

```diff
@@ -60,14 +60,21 @@ if (!haveIntl) {
 
   // Check with toLocaleString
   var localeString = dtf.format(date0);
-  assert.equal(localeString, 'Jan 70');
+  assert.equal(localeString, common.engineSpecificMessage({
+    v8: 'Jan 70',
+    chakracore: '\u200EJan\u200E \u200E70'
```

Comment:
This issue is known and being discussed at ChakraCore and TC39:

https://github.com/Microsoft/ChakraCore/issues/599
https://github.com/tc39/ecma402/issues/28

Patching it enables rest of the test to run and gives us more test coverage.